packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -26,7 +26,16 @@ module.exports = (resolve, rootDir, isEjecting) => {
     setupFiles: [resolve('config/polyfills.js')],
     setupTestFrameworkScriptFile: setupTestsFile,
     testPathIgnorePatterns: [
-      '<rootDir>[/\\\\](build|docs|node_modules|scripts)[/\\\\]',
+      // Ignore the following directories:
```

Comment:
I figured there were enough things here to warrant some comments. I couldn't figure out why `docs` is in there though.